net/dns/quad100.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package dns
+
+import (
+	"bytes"
+	"context"
+	"net"
+	"net/netip"
+	"time"
+)
+
+type ManagerConn struct {
+	Ctx        context.Context
+	DnsManager *Manager
```

Comment:
DNSManager in Go style: https://go.dev/wiki/CodeReviewComments#initialisms

but can it just be unexportes as "mgr" instead?